app/models/damage.rb

```diff
@@ -0,0 +1,24 @@
+class Damage < ActiveRecord::Base
```

Comment:
I see - that way people can reserve an item for a future time when it is anticipated it will have been repaired.

Perhaps, rather than using a reservation to accomplish this, the individual Damage objects could have fields for tracking their anticipated resolution time.